GitUI/Editor/FindAndReplaceForm.cs

```diff
@@ -401,11 +408,36 @@ protected override void Dispose(bool disposing)
 
             base.Dispose(disposing);
         }
+
+        internal TestAccessor GetTestAccessor() => new TestAccessor(this);
+
+        internal readonly struct TestAccessor
+        {
+            private readonly FindAndReplaceForm _control;
+
+            public TestAccessor(FindAndReplaceForm control)
+            {
+                _control = control;
+            }
+
+            public TextEditorSearcher Search => _control._search;
+            public TextBox TxtLookFor => _control.txtLookFor;
+            public CheckBox ChkMatchCase => _control.chkMatchCase;
+            public CheckBox ChkMatchWholeWord => _control.chkMatchWholeWord;
+
+            public TextEditorControl Editor
+            {
+                set
+                {
+                    _control.Editor = value;
+                }
+            }
```

Comment:
Set-only properties are a code smell, in this case please use the same approach as for the other members:
```suggestion
            public TextEditorControl Editor => _control.Editor;
```

Reply:
Suggested code introduces a get-only property which is not needed. The reason this property is set-only is that the underlying _control.Editor is a set-only property (test accessor is supposed to be dumb as you explained and should just elevate access level). I agree that it smells :) my first thought was not to change the original code. I'll convert that underlying FindAndReplaceForm.Editor to be a method named SetEditor if you don't mind and it'll become a method here as well.

Reply:
Done